IngresseSDK/Network/Network.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Network.swift
+//  Alamofire
+//
```

Comment:
Geralmente a gente deixa assim no cabeçalho:

```suggestion
//
//  Copyright © 2020 ingresse. All rights reserved.
//
```